packages/vkui/src/components/CustomSelect/CustomSelect.tsx

```diff
@@ -148,6 +158,14 @@ export interface SelectProps extends NativeSelectProps, FormFieldProps, TrackerO
   onClose?: VoidFunction;
   onOpen?: VoidFunction;
   icon?: React.ReactNode;
+  /**
+   * Кастомная кнопка для очистки значения
+   */
+  clearButton?: React.ReactNode;
```

Comment:
```suggestion
  ClearButton?: React.ComponentType<CustomSelectClearButtonProps>;
```

нам нужен именно конструктор компонента, т.к. нам надо пробросить `onClear()` как в `CustomSelectClearButton`, иначе не очистим

Reply:
А зачем нам это понимать?)

Reply:
Если я передам `clearButton={<MyButton>x</MyButton>}`, то как нам внутри `CustomSelect` отследить, что нажали на крестик?

Reply:
Плохой DX

И это нормально просить предоставить нужный нам интерфейс. И в коде будет что-то типа такого

```tsx
const Comp = ({ ClearButton = CustomSelectClearButton }) => {
  const clearButton = <CustomSelectClearButton className={...} onClear={() => setNativeSelectValue('')} />
  
  return <div>{clearButton}</div>;
};
```